plugin/src/Reconciler/diff.lua

```diff
@@ -113,6 +113,13 @@ local function diff(instanceMap, virtualInstances, rootId)
 			local childId = instanceMap.fromInstances[childInstance]
 
 			if childId == nil then
+				if childInstance.Name == "__Rojo_SessionLock" then
```

Comment:
I wonder if we should ignore everything that has `Archivable = false`. This seems like a fine workaround for now probably.

Reply:
I can't think of a case where you'd want to have Rojo delete your temp instances, yeah. Good point! Should I make the switch?

Reply:
Yeah, let's do that!

Reply:
This would then be considered a breaking change (for 8.0.0), right?

Reply:
Hm. It's certainly a change in behavior, but what would it break?

Reply:
I don't suspect it would break anything, but I wonder if it still classes as one!

Reply:
I'd say no, as it doesn't break existing workflows. If anything it enables them as Rojo will no longer kill your temp files.